pkg/helper/pipelinerun/description/description.go

```diff
@@ -24,6 +24,12 @@ const templ = `{{decorate "bold" "Name"}}:	{{ .PipelineRun.Name }}
 {{decorate "bold" "Service Account"}}:	{{ .PipelineRun.Spec.ServiceAccountName }}
 {{- end }}
 
+{{decorate "timeout" ""}} {{decorate "underline bold" "Timeout\n"}}
```

Comment:
This should probably be similar to the following for Namespace:

```
{{decorate "bold" "Namespace"}}:	{{ .TaskRun.Namespace }}
```

So something like:

```
{{decorate "bold" "Timeout"}}:	{{ .PipelineRun.Spec.Timeout.Duration.String }}
```

Reply:
Updated